packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -118,6 +120,17 @@ export const CommandMenu = () => {
       ],
     },
     limit: 3,
+    onCompleted: (data) => {
```

Comment:
We should put all the onCompleted in useEffects, because onCompleted isn't retriggered when we're updating the cache with optimistic effects.

Reply:
Hey @lucasbordeau Adding a `useEffect` just caused too many rerenders, please check out the change I went with.